docs/external-configuration/_index.md

```diff
@@ -0,0 +1,492 @@
+---
+title: External configuration for Vault
+shorttitle: External Vault configuration
+weight: 30
+---
+
+In addition to the [standard Vault configuration](https://www.vaultproject.io/docs/configuration/index.html), the operator and CLI can continuously configure Vault using an external YAML/JSON configuration.
+<!-- FIXME Explain how to use this feature and why is it good. CI/CD/gitops for vault? -->
```

Comment:
This is good because you can configure Vault in a declarative and automated way with YAML files which is the de-facto standard in the Kubernetes world.